### PR TITLE
Issue 161 Fix

### PR DESCRIPTION
**Pull request by btakita**
Here's a fix for Issue #161.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,6 +2,7 @@
 *.rbc
 .bundle
 .config
+.idea
```
Comment (vassilevsky) [inline .gitignore]: RubyMine?

File: ext/RMagick/extconf.rb [hunk 2]
```diff
@@ -3,7 +3,7 @@
 
 module RMagick
   class Extconf
-    require 'rmagick/version'
+    require "#{File.dirname(__FILE__)}/../../lib/rmagick/version"
```
Comment (mockdeep) [inline ext/RMagick/extconf.rb]: Do we need a `File.join` here to be compatible with windows?

File: ext/RMagick/extconf.rb [hunk 3]
```diff
@@ -130,6 +130,11 @@ def configure_compile_options
 
     end
 
+    def cant_install_rmagick?(version)
```
Comment (vassilevsky) [inline ext/RMagick/extconf.rb]: It would be more natural to define a positive method (`can_install?`) and then negate its usage if necessary.

**Comment by vassilevsky** [comment]
Ugh. I'm struggling to understand how this entire logic works. Which prerequisites are necessary and which are not? Could it be written so that this is more obvious?

File: .gitignore (hunk 1 above)
Comment (btakita) [inline .gitignore]: Yes.

File: ext/RMagick/extconf.rb (hunk 2 above)
Comment (btakita) [inline ext/RMagick/extconf.rb]: I added `File.join`

File: ext/RMagick/extconf.rb (hunk 3 above)
Comment (btakita) [inline ext/RMagick/extconf.rb]: I renamed the method to `can_install?`

**Comment by btakita** [comment]
I agree that the logic is convoluted. I just wanted a quick solution to installing rmagick on debian. Can we make the refactoring a different pull request?

File: .gitignore (hunk 1 above)
Comment (btakita) [inline .gitignore]: Yes.

File: ext/RMagick/extconf.rb (hunk 2 above)
Comment (btakita) [inline ext/RMagick/extconf.rb]: I added `File.join`

**Comment by btakita** [comment]
I'm able to install the `master` branch on `gemhome/rmagick`, so I'm closing this pull request.

**Pull request closed without merging**